src/OpenIddict.Core/OpenIddictManager.cs

```diff
@@ -31,12 +35,73 @@ public OpenIddictManager([NotNull] IServiceProvider services)
         public virtual HttpContext Context { get; }
 
         /// <summary>
+        /// Gets the Identity options associated with the current manager.
+        /// </summary>
+        public virtual IdentityOptions Options { get; }
+
+        /// <summary>
         /// Gets the store associated with the current manager.
         /// </summary>
         public virtual new IOpenIddictStore<TUser, TApplication> Store {
             get { return base.Store as IOpenIddictStore<TUser, TApplication>; }
         }
 
+        public virtual async Task<ClaimsIdentity> CreateIdentityAsync(TUser user, IEnumerable<string> scopes) {
```

Comment:
@damccull this new method should make adding custom claims to the access/identity tokens a bit easier :smile: